util/timer.h

```diff
@@ -111,7 +112,7 @@ class Timer {
     if (running_) {
       return false;
     }
-
+    WaitForShuttingDownIfNecessary();
```

Comment:
I don't foresee a practical need to have Start block on a pending shutdown. It could just return false instead. If the user wants Start to be guaranteed, Shutdown should have returned without an intervening Start.

Reply:
But almost all code that uses Timer doesn't check the return value.